SMLHelper/Assets/Spawnable.cs

```diff
@@ -98,9 +101,13 @@ protected Spawnable(string classId, string friendlyName, string description)
             CorePatchEvents += () =>
             {
                 PrefabHandler.RegisterPrefab(this);
-                SpriteHandler.RegisterSprite(TechType, GetItemSprite());
 
-                if(!SizeInInventory.Equals(defaultSize))
+#if SN1
```

Comment:
```suggestion
#if SUBNAUTICA
```
found another one I missed before